recipe-page/index.html

```diff
@@ -48,13 +49,14 @@ <h2>Steps:</h2>
                     </ol>
                 </div>
                 <div class="similar">
-                    <h2>Other Recipes to try:</h2>
-                    <ul>
+                    <p id="other">See other recipes?</p>
```

Comment:
Order of operations is wrong here, it starts shown, but the button says "Show"
![Screen Shot 2023-07-28 at 10 38 30 AM](https://github.com/RachelGreenwood/TechtonicaAssignments/assets/892249/6a8fc507-a7fa-4f51-83ff-899b165a2733)